earthly/spectral/Earthfile

```diff
@@ -3,29 +3,18 @@ VERSION 0.8
 # cspell: words ruleset
 
 spectral-base:
-    FROM stoplight/spectral
+    FROM stoplight/spectral:6.13.1
     WORKDIR /work
-    COPY . .
-    RUN chmod +x ./minify-json.sh
-    SAVE ARTIFACT minify-json.sh
 
 BUILD_SPECTRAL:
     FUNCTION
-
-    # Specify what file type to lint 
-    ARG file_type = "json"
+    
+    # Directory to lint
     ARG dir = .
-    ARG src = .
     # Rule set for spectral
-    ARG rule_set=.spectral.yml
-
-    COPY $src .
-
-    COPY +spectral-base/minify-json.sh minify-json.sh
-    # If file type is json, minify the JSON
-    RUN ./minify-json.sh
+    ARG rule_set = .spectral.yml
 
     RUN spectral \
         lint \
         $dir/"**/*.{yml,json}" \
```

Comment:
This will cause the .spectral.yml file to get linted, which does have tabs in it.
We should restrict this to simply linting json.  There is no need for us to lint yaml openapi docs.